<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trane - Sparky</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/resource.qrc">
    <normaloff>:/lightning.png</normaloff>:/lightning.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>HEX REQUEST</string>
    </property>
    <widget class="QLineEdit" name="request">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>VIEWER</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-2</y>
        <width>539</width>
        <height>293</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="hyperTerm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>300</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ASCII </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hyperTerm2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MACRO MANAGER</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>541</width>
        <height>321</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>MACRO NAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>HEX VALUE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TIME OUT (msec)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RESPONSE</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tempEdit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>-20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>BATCH MANAGER</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-1</y>
        <width>538</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>RESPONSE</string>
    </property>
    <widget class="QLineEdit" name="response">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>SELECT HEX MACRO</string>
    </property>
    <widget class="QComboBox" name="selectMacro">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>SELECT BATCH</string>
    </property>
    <widget class="QComboBox" name="selectBatch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>RESPONSE TIME (msec)</string>
    </property>
    <widget class="QLineEdit" name="delay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionClean_Log_Screen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClean_Log_Screen">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
   </property>
   <property name="text">
    <string>Clean Log Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/disconnect.png</normaloff>:/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
